### PR TITLE
All application scripts updated to use miles-guess

### PR DESCRIPTION
**Pull request by jsschreck**
Overall I mostly updated the model, importing it from our miles-guess repo. The trainer, evaluation, and active_training scripts each had portions changed to reflect the updated merge.

### CODE REVIEW

File: applications/active_training.py [hunk 1]
```diff
@@ -136,7 +140,7 @@ def launch_pbs_jobs(
         #PBS -e {os.path.join(save_path, "out")}
 
         source ~/.bashrc
-        conda activate ptype
+        conda activate guess-casper
```
Comment (djgagne) [inline applications/active_training.py]: Is there a way to make the environment a configuration option rather than be hard-coded?

File: applications/evaluate_mlp.py [hunk 2]
```diff
@@ -112,7 +145,7 @@ def evaluate(conf, reevaluate=False):
     plot_confusion_matrix(
         data,
         labels,
-        normalize=True,
+        normalize="true",
```
Comment (djgagne) [inline applications/evaluate_mlp.py]: Why change from bool to string? Bool is more appropriate for True/False flags.

File: applications/train_mlp.py [hunk 3]
```diff
@@ -5,6 +5,340 @@
...
+from bridgescaler import save_scaler
+
+
+warnings.filterwarnings("ignore")
+
+
+logger = logging.getLogger(__name__)
+
+
+class Objective(BaseObjective):
```
Comment (djgagne) [inline applications/train_mlp.py]: Can you move the Objective class to a module file in the ptype directory and import it rather than having it all in the trainer script?

**Review by djgagne — CHANGES_REQUESTED** [review]
I have some code relocation changes to request and a couple of small clarifications. Otherwise it looks like the main refactor is successful.

File: applications/active_training.py (hunk 1 above)
Comment (jsschreck) [reply applications/active_training.py]: I can make these all config options?

File: applications/evaluate_mlp.py (hunk 2 above)
Comment (jsschreck) [reply applications/evaluate_mlp.py]: There is a bug somewhere in the data loader script ... I havent dug it up yet but will fix this on the next commit.

File: applications/train_mlp.py (hunk 3 above)
Comment (jsschreck) [reply applications/train_mlp.py]: Yes!

File: applications/evaluate_mlp.py (hunk 2 above)
Comment (jsschreck) [reply applications/evaluate_mlp.py]: Ok the root of this is due to passing normalize directly into the sklearn.metrics.confusion_matrix, which wants None or 'true'. I've just updated our script so that its True and not 'true' and handle it in the method in plotting.py

**Review by djgagne — APPROVED** [review]
Looks good to me.
